examples/bert/bert_finetune_glue.py

```diff
@@ -18,11 +18,11 @@
 import datasets
 import keras_tuner
 import tensorflow as tf
-import tensorflow_text as tf_text
 from absl import app
 from absl import flags
 from tensorflow import keras
 
+import keras_nlp
```

Comment:
Why do you need to import from the parent package? Shouldn't we be importing from relative subfolders? The style guide seems to imply this.

Reply:
For code inside the library (inside `keras_nlp/`), we should use relative imports.

But `examples/` is not part of the library, this is example code using the library as any client would! I can add some language to clear that up to the style guide. We should never be shipping code in `examples/` or anything outside of `keras_nlp/` really.